packages/dashboard/src/PanelManager.ts

```diff
@@ -187,13 +187,20 @@ class PanelManager {
     C extends ComponentType<P> = ComponentType<P>
   >(types: PanelComponentType<P, C>[]): PanelComponent<P> | undefined {
     return this.getLastUsedPanel(panel =>
-      types.some(
-        type =>
-          panel instanceof type ||
+      types.some(type => {
+        let instanceofType;
+        try {
+          instanceofType = panel instanceof type;
+        } catch (e) {
+          instanceofType = false;
+        }
```

Comment:
Add a comment mentioning that the catch block is needed due to safari behavior

Reply:
Oops didn't notice this was in draft state. Still add a comment for the final though

Reply:
It's interesting - in this case, `type` that is passed in is a typeof `"object"`, so I don't think `instanceof` would _ever_ be true, and it's kind of surprising that Chrome/Firefox don't throw an error here. We probably shouldn't be checking `instanceof` on just `type` if it's a wrapped component.
So I think to be more correct, we could just do something like:
```
return (isWrappedComponent(type) && panel instanceof type.WrappedComponent) || (!isWrappedComponent(type) && panel instanceof type);
```